theano/configdefaults.py

```diff
@@ -126,6 +126,12 @@ def __str__(self):
     BoolParam(False, allow_override=False),
     in_c_key=False)
 
+AddConfigVar(
```

Comment:
I forgot, this should go in the doc in the file doc/library/config.txt.

It would be good to also include a paragraph about it in the file doc/tutorial/profiling.txt

Can you do that too?

thanks